content/topics/How-to-ask-for-help-with-your-code/_index.md

```diff
@@ -9,16 +9,24 @@ Practice asking good questions. It's like a superpower. If you are easy to help
 
 ### These also include:
 
-#### how to ask good questions on slack:
+#### Where to ask good questions
 
-* all Slack questions should be addressed on the slack **Tech Assist** channel.
- - This helps you get to a broad audience from the tech department (.e.g Recruits, Managers and Alumni).
- - Please don't send screenshots what you can do is use backtick (` `) characters to represent a code form.
- - Write full error messages. (Summarise if you can and make sure you understand the error before sending it)
+ - Slack
+ - [Stackoverflow](https://stackoverflow.com/help/how-to-ask) == TRUELY **AMAZING**
+ - [Google](https://www.google.com)
+
+#### Characteristics of a good question:
+
+ - All Slack questions should be addressed on the slack **Tech Assist** channel.
+ - Asking on the channel helps you get to a much bigger audience from the Tech department (.e.g Recruits, Managers and Alumni).
+ - Be clear and not ambiguous. Your "helpers" should be able to understand the problem while reading an error message.
+ - Please <span style="color:red">don't send screenshots</span>, what you can do is use [backtick](https://slack.com/intl/en-za/help/articles/202288908-format-your-messages) symbols to represent in code form.
+ - Write full error messages.
+ - Be short and meaningful. Error message should contain necessary information. Most of the time user is not willing to read a long story.
```

Comment:
In general, if your question text contains code or error message those things need to be formatted properly. It needs to not look like a jumbled mess of words and symbols. Different platforms have different ways of pulling this off. Slack uses the backtick symbol. And Stackoverflow does it it's own way. Be aware of the tools ond conventions of the platform you are using, then follow those conventions. 

As a general rule of thumb: If you really want help then make it as easy as possible for people to help you. Empathise with the person who will help you. If they are left guessing or they have to ask a million follow up questions then that means you asked a bad-quality question.